resources/js/views/general/Tag.vue

```diff
@@ -21,8 +19,7 @@
 
             <!-- Image is available for tagging -->
             <div v-else>
-
-                <div v-for="photo in photos" class="mb2">
+                <div v-for="photo in photos" :key="photo.id + photo.filename" class="mb2">
```

Comment:
`:key="photo.id"` is enough

Reply:
`photo.id` is just a number like an index, we should not use an index as a key.

Reply:
Actually, photo.id is unique key. It is not `photos.map( (item, **index**) =>`

Reply:
Updated. But you should read more about key in Vue.